networks/ibc/README.md

```diff
@@ -0,0 +1,7 @@
+## Local IBC Test Cluster
```

Comment:
Add more detail here.  Using an ibc cluster can be confusing even for someone knowing what they are doing.  We are creating this cluster for developers that might be doing using ibc for the first time.  Maybe some transfer example too?

Reply:
Good idea. Added some more documentation and included an example.

Reply:
The changes look great! Went through them locally and everything worked without.